### PR TITLE
Dev environment for java developers

### PR DESCRIPTION
**Pull request by octonato**

### CODE REVIEW

File: docs-source/docs/modules/microservices-tutorial/pages/dev-env.adoc [hunk 1]
```diff
@@ -23,16 +24,23 @@ We also provide downloadable versions of the example at different phases of comp
...
+[.group-java]
+Download and install https://www.jetbrains.com/idea/download/[IntelliJ {tab-icon}, window="tab"].
+
+[.group-scala]
 To install IntelliJ and prepare to develop the Scala example, follow these steps:
- 
+
+[.group-scala]
 . Download and install https://www.jetbrains.com/idea/download/[IntelliJ {tab-icon}, window="tab"].
 . From the *Preferences* menu, add the *Scala* plugin.
```
Comment (patriknw) [inline docs-source/docs/modules/microservices-tutorial/pages/dev-env.adoc:35]: does such group continue until the next empty newline?

**Review by patriknw — APPROVED** [review]
LGTM

File: docs-source/docs/modules/microservices-tutorial/pages/dev-env.adoc (hunk 1 above)
Comment (octonato) [reply docs-source/docs/modules/microservices-tutorial/pages/dev-env.adoc:35]: I don't know the rule, but in that case, it wraps the whole list. So, yes, both lines are suppressed in "java mode"

**Review by rstento — APPROVED** [review]
LGTM!

**Comment by ignasi35** [comment]
this build has a dirty cache in travis. You can either rebase over `main` once https://github.com/akka/akka-platform-guide/pull/313 is merged, or close the PR and create a new one.

**Pull request merged**